modules/serverless-kafka-source-autoscale-keda.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * eventing/event-sources/serverless-kafka-developer-source.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-source-autoscale-keda_{context}"]
+= Configuring KEDA autoscaling for KafkaSource
+
+You can configure Knative Eventing sources for Apache Kafka (KafkaSource) to be autoscaled using the Custom Metrics Autoscaler Operator, which is based on the Kubernetes Event Driven Autoscaler (KEDA).
+
+:FeatureName: Configuring KEDA autoscaling for KafkaSource
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource are installed on your cluster.
+
+.Procedure
+
+. In the `KnativeKafka` custom resource, enable KEDA scaling:
++
+.Example YAML
+[source,yaml]
+----
```

Comment:
There's "config" missing in the spec, the example should be like this:
```
apiVersion: operator.serverless.openshift.io/v1alpha1
kind: KnativeKafka
metadata:
  name: knative-kafka
  namespace: knative-eventing
spec:
  config:
    kafka-features:
      controller-autoscaler-keda: enabled
```

Reply:
Thanks, fixed!